rmgpy/molecule/moleculeTest.py

```diff
@@ -1276,6 +1278,123 @@ def testSaturateAromaticRadical(self):
+20 C u0 p0 c0 {14,B} {15,B} {30,S}
+21 H u0 p0 c0 {17,S}
+22 H u0 p0 c0 {9,S}
+23 H u0 p0 c0 {10,S}
+24 H u0 p0 c0 {18,S}
+25 H u0 p0 c0 {11,S}
+26 H u0 p0 c0 {12,S}
+27 H u0 p0 c0 {19,S}
+28 H u0 p0 c0 {13,S}
+29 H u0 p0 c0 {14,S}
+30 H u0 p0 c0 {20,S}
+31 H u0 p0 c0 {15,S}
+32 H u0 p0 c0 {16,S}
+""")
+        perylene2 = Molecule().fromSMILES('c1cc2cccc3c4cccc5cccc(c(c1)c23)c54')
+        for isomer in generateAromaticResonanceIsomers(perylene2):
+            if perylene.isIsomorphic(isomer):
+                break
+        else:  # didn't break
+            self.fail("{} isn't isomorphic with any aromatic forms of {}".format(
+                            perylene.toSMILES(),
+                            perylene2.toSMILES()
+                        ))
+
+    def testFusedAromatic2(self):
+        """Test we can make aromatic naphthalene from both adjlist and SMILES"""
+        naphthalene = Molecule().fromAdjacencyList("""
+1  C u0 p0 c0 {2,B} {3,B} {4,B}
+2  C u0 p0 c0 {1,B} {5,B} {6,B}
+3  C u0 p0 c0 {1,B} {8,B} {13,S}
+4  C u0 p0 c0 {1,B} {9,B} {14,S}
+5  C u0 p0 c0 {2,B} {10,B} {17,S}
+6  C u0 p0 c0 {2,B} {7,B} {18,S}
+7  C u0 p0 c0 {6,B} {8,B} {11,S}
+8  C u0 p0 c0 {3,B} {7,B} {12,S}
+9  C u0 p0 c0 {4,B} {10,B} {15,S}
+10 C u0 p0 c0 {5,B} {9,B} {16,S}
+11 H u0 p0 c0 {7,S}
+12 H u0 p0 c0 {8,S}
+13 H u0 p0 c0 {3,S}
+14 H u0 p0 c0 {4,S}
+15 H u0 p0 c0 {9,S}
+16 H u0 p0 c0 {10,S}
+17 H u0 p0 c0 {5,S}
+18 H u0 p0 c0 {6,S}
+""")
+        naphthalene2 = Molecule().fromSMILES('C1=CC=C2C=CC=CC2=C1')
+        for isomer in generateAromaticResonanceIsomers(naphthalene2):
+            if naphthalene.isIsomorphic(isomer):
+                break
+        else:  # didn't break
+            self.fail("{} isn't isomorphic with any aromatic forms of {}".format(
+                            naphthalene.toSMILES(),
+                            naphthalene2.toSMILES()
+                        ))
+
+    def testFusedAromatic3(self):
+        """Test we can make aromatic pyrene_rad from both adjlist and SMILES"""
+        pyrene_rad = Molecule().fromAdjacencyList("""
+multiplicity 2
+1  C u0 p0 c0 {2,B} {3,B} {5,B}
+2  C u0 p0 c0 {1,B} {4,B} {6,S}
+3  C u0 p0 c0 {1,B} {8,B} {9,B}
+4  C u0 p0 c0 {2,B} {10,B} {11,S}
+5  C u0 p0 c0 {1,B} {7,B} {15,S}
+6  C u0 p0 c0 {2,S} {12,S} {16,D}
+7  C u0 p0 c0 {5,B} {13,B} {17,S}
+8  C u0 p0 c0 {3,B} {13,B} {19,S}
+9  C u0 p0 c0 {3,B} {10,B} {20,S}
+10 C u0 p0 c0 {4,B} {9,B} {21,S}
+11 C u1 p0 c0 {4,S} {14,S} {22,S}
+12 C u0 p0 c0 {6,S} {14,D} {24,S}
+13 C u0 p0 c0 {7,B} {8,B} {18,S}
+14 C u0 p0 c0 {11,S} {12,D} {23,S}
+15 C u0 p0 c0 {5,S} {16,D} {25,S}
+16 C u0 p0 c0 {6,D} {15,D}
+17 H u0 p0 c0 {7,S}
+18 H u0 p0 c0 {13,S}
+19 H u0 p0 c0 {8,S}
+20 H u0 p0 c0 {9,S}
+21 H u0 p0 c0 {10,S}
+22 H u0 p0 c0 {11,S}
+23 H u0 p0 c0 {14,S}
+24 H u0 p0 c0 {12,S}
+25 H u0 p0 c0 {15,S}
+""")
+        pyrene_rad2 = Molecule().fromSMILES('[C]1C=C2C=CC=C3C=CC4=CC=CC=1C4=C23')
+        for isomer in pyrene_rad2.generateResonanceIsomers():
```

Comment:
Should this be `generateAromaticResonanceIsomers(pyrene_rad2)`?

On a related note, RMG goes crazy trying to generate resonance isomers for the kekulized form of this pyrene radical, moving the radical all over the molecule. However, this radical can't shift because it's in an sp2 orbital that's not in the same plane as the delocalized pi system. Another issue to add to the aromaticity list...